src/components/typography.tsx

```diff
@@ -85,3 +86,14 @@ export const A = ({ children, className, href, ...props }: AnchorProps) =>
       {children}
     </Link>
   )
+
+type ImgProps = ImgHTMLAttributes<HTMLImageElement>
+
+export const Img = ({ className, alt, src, ...props }: ImgProps) => (
+  <img
+    alt={alt}
+    src={src && parseImageUrl(src)}
```

Comment:
We should consider supporting `srcset` here since Strapi provides various sizes of images.